src/main/java/io/confluent/connect/jdbc/source/TableMonitorThread.java

```diff
@@ -130,6 +130,7 @@ public void run() {
   }
 
   public void shutdown() {
+    log.info("Shutting down TableMonitorThread");
```

Comment:
Since this is INFO level, how about:
```suggestion
    log.info("Shutting down thread monitoring tables.");
```